lib/vagrant-service-manager/action/setup_network.rb

```diff
@@ -10,11 +10,11 @@ def initialize(app, env)
         end
 
         def call(env)
-          @app.call(env)
-
-          if SUPPORTED_BOXES.include? @machine.guest.capability(:os_variant)
+          if SUPPORTED_BOXES.include? @machine.config.vm.box.sub('projectatomic/', '')
```

Comment:
where is the ' @app.call(env)' call? You need to continue the chain, right? Why did you do it on the up chain anyways?